### ISSUE TITLE
RFC-0153 Emoji ID secret key should also be stored in stage 1b

### ISSUE THREAD

**Issue by stringhandler** [tari-project/rfcs#79]
The emoji id for a wallet should not change when a restore is done. If the seed words are removed, then the restored wallet will have a different emoji id. 

I suggest storing the secret key for the emoji id in stage 1b, or at least mentioning that it should be explicitly removed

**Comment by CjS77** [comment]
Correct. To avoid leaking all future funds, the trade-off is that you import the old UTXOs into a new wallet. You're not really restoring the wallet _per sė_. We should update the text to make this clear.
